sstmac/hardware/snappr/snappr_nic.cc

```diff
@@ -80,7 +80,7 @@ SnapprNIC::SnapprNIC(uint32_t id, SST::Params& params, Node* parent) :
   SST::Params inj_params = params.find_scoped_params("injection");
 
   //configure for a single port for now
-  outports_.reserve(1);
+  outports_.resize(1);
```

Comment:
Resize outports_ to num_ports, that way when you change the number of ports in the future the loop isn't broken.

Reply:
Changed